docs/source/getting-started/index.rst

```diff
@@ -0,0 +1,107 @@
+Getting Started
+===============
+
+.. toctree::
+    :hidden:
+
+    installing-idom
+    running-idom
+
+.. dropdown:: :octicon:`bookmark-fill;2em` What You'll Learn
+    :color: info
+    :animate: fade-in
+    :open:
+
+    .. grid:: 2
+
+        .. grid-item-card:: :octicon:`tools` Installing IDOM
+            :link: installing-idom
+            :link-type: doc
+
+            Learn how IDOM can be installed in a variety of different ways - with different web
+            servers and even in different frameworks.
+
+        .. grid-item-card:: :octicon:`play` Running IDOM
+            :link: running-idom
+            :link-type: doc
+
+            See the ways that IDOM can be run with servers or be embedded in existing
+            applications.
+
+The fastest way to get started with IDOM is to try it out in a Juptyer Notebook. If you
+want to use a Notebook to work through the examples shown in this documentation, you'll
+want to replace calls to ``idom.run(App)`` with a line at the end of each cell that
+constructs the ``App()`` in question. If that doesn't make sense, the introductory
+notebook linked below will demonstrate how to do this:
+
+.. card::
+    :link: https://mybinder.org/v2/gh/idom-team/idom-jupyter/main?urlpath=lab/tree/notebooks/introduction.ipynb
+
+    .. image:: _static/idom-in-jupyterlab.gif
+        :scale: 72%
+        :align: center
+
+
+Section 1: Installing IDOM
+--------------------------
+
+The next fastest option is to install IDOM with ``pip``:
```

Comment:
I'm not really sure this is the best place for that kind of note since this section is really just trying to help the user try out IDOM as quickly and easily as possible. If the user is interested in learning more, they can following the "Read More" link where they'll see where they can learn about using IDOM with Django or they could search for the term "django" to find it.